package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -681,6 +737,22 @@ def _read_next_timestep(self):
             coord_cols.extend(image_cols)
 
         ids = "id" in attr_to_col_ix
+
+        # Create the data arrays for additional attributes which will be saved
+        # under ts.data
+        if self._additional_columns is True:
+            # Parse every column that is not already parsed
+            # elsewhere (total \ parsable)
+            additional_keys = set(attrs).difference(self._parsable_columns)
+        elif self._additional_columns:
+            additional_keys = \
+                [key for key in self._additional_columns if key in attrs]
```

Comment:
Could you warn if a key specified in as kwarg is not present in attrs? This way users will know if attributes specified are not being read in. It will be a bit noisy, but I think its worth the clarity.

Reply:
Should be implemented now :).